src/layouts/faqs.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+class Faqs extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      question: []
+    }
+
+  }
+
+
+
+  componentDidMount() {
```

Comment:
Now that you have taste of using ‘componentDidMount’ can we update this to just import flat file and baked in rather than requesting for FAQ